tests/pre_v6_test.py

```diff
@@ -0,0 +1,88 @@
+from pyteal import *
```

Comment:
Are these tests passing? If so, something is up with my other tests which generate more random slot assignments.